### PR TITLE
Fixing issue with relative paths starting with tilde

### PR DESCRIPTION
**Pull request by jteheran**
This fix is for an error reported by an FNAL user:
File not found when a relative path was provided for a cert and/or a key file preceded by an equal sign

`--cert=~/path/to/cert`

Also removed the fallback path for cert and key: X509_USER_CERT and X509_USER_KEY, ~/.globus/usercert.pem and ~/.globus/userkey.pem

### CODE REVIEW

File: osgpkitools/incommon_request.py [hunk 1]
```diff
@@ -204,10 +203,13 @@ def parse_args():
...
     arguments.update({'login': args.login})
     
-    if args.usercert and args.userprivkey:
-        usercert, userkey = utils.find_user_cred(args.usercert, args.userprivkey)
-        arguments.update({'usercert': usercert})
-        arguments.update({'userprivkey': userkey})
+    if not args.usercert or not args.userprivkey:
+        raise InsufficientArgumentException("InsufficientArgumentException: " + \
+                                            "Please provide certificate(-c, --cert) and key(-k, --pkey) files\n")
+
```
Comment (DrDaveD) [inline osgpkitools/incommon_request.py]: Remember to remove the square brackets around -c cert -k pkey in the usage in incommon-request.py, and also in README.md.

Comment (jteheran) [reply osgpkitools/incommon_request.py]: Removed.

File: osgpkitools/utils.py [hunk 2]
```diff
@@ -140,35 +140,27 @@ def check_permissions(path):
...
     """
-    
-    # list of cert/key pairs to try
-    input_pairs = [(usercert, userkey),
-                   ((os.environ.get('X509_USER_CERT'), os.environ.get('X509_USER_KEY'))),
-                   (os.path.expanduser('~/.globus/usercert.pem'), os.path.expanduser('~/.globus/userkey.pem'))]
-    cert_key_pairs = [t for t in input_pairs if None not in t] # remove undefined pairs for an improved err msg below
 
+    cert = os.path.expanduser(usercert)
+    key = os.path.expanduser(userkey)
```
Comment (DrDaveD) [inline osgpkitools/utils.py]: Now that there's no fallback, I don't actually see any need for the expanduser calls.  Can you think of any example cases where it makes a difference?

Comment (jteheran) [reply osgpkitools/utils.py]: The case that Joe B. reported with long option, equal sign "=" and the path starting with tilde:

`osg-incommon-cert-request -T -u fnalclient@fnal.gov --cert=~certs/jteheran_cert.pem --pkey=~/certs/jteheran_key.pem`

Path cannot be found if I don't do the expanduser(path)

Comment (DrDaveD) [reply osgpkitools/utils.py]: Oh I was thinking there wasn't supposed to be an equal there but I got it confused with the short version, a separate issue, ok.

**Pull request merged**